voyager-navigator/src/commonMain/kotlin/cafe/adriel/voyager/navigator/Navigator.kt

```diff
@@ -173,6 +173,11 @@ public class Navigator @InternalVoyagerApi constructor(
         }
     }
 
+    internal fun popRecursively() {
+        if (pop()) return
+        parent?.popRecursively()
+    }
```

Comment:
Can you make this function public annotated with  `@InternalVoyagerApi`?

Reply:
Hey, @DevNatan are you going to fix this ?